apps/web/src/hooks/use-space-documents.ts

```diff
@@ -68,9 +69,11 @@ export const useSpaceDocuments: UseDocuments = ({
   };
 
   const parsedData = response?.data.map((document: Document) => {
+    const { person } = usePersonById(document.creatorId);
     return {
       ...document,
       badges: getDocumentBadges(document),
+      creator: person,
     };
   });
```

Comment:
If you want the creator to be included in the document type, why not add it on the server side?